site/content/en/references/kustomize/kustomization/helmcharts/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Helm Charts"
+linkTitle: "Helm Charts"
+type: docs
+weight: 6
+description: >
+Helm chart inflation generator
```

Comment:
indentation is missing here, which I believe is preventing the page from being built (it's not in the preview site)

Reply:
Thank you for your help! This is fixed now